install.sh

```diff
@@ -3,6 +3,9 @@
 echo Installing byzanz
 sudo apt-get install autoconf byzanz
 
+echo Installing missing packages
+sudo apt-get install automake libtool
```

Comment:
could you move this inside `lolilolicon/xrectsel` section below as it `xrectsel` requirement, not `record-gif.sh`.